config/initializers/active_admin.rb

```diff
@@ -331,14 +331,7 @@
   # By default, Active Admin uses Sprocket's asset pipeline.
   # You can switch to using Webpacker here.
   #
-  config.use_webpacker = true
-end
-
-# Print stylesheet is no longer a separate stylesheet in upstream so we can
-# delete this when the next version bump is released but for now it prevents
-# webpacker issues.
-Rails.application.config.after_initialize do
-  ActiveAdmin.application.stylesheets.delete("active_admin/print.css")
+  # config.use_webpacker = true
```

Comment:
do we need this?

Reply:
No that's why it's commented out now but it (commented version) is part of the default generated config file so I've just reverted it to that.